File: simplesearchview/src/main/java/com/ferfalk/simplesearchview/SimpleTextWatcher.java
package com.ferfalk.simplesearchview;

import android.text.Editable;
import android.text.TextWatcher;

/**
 * @author Fernando A. H. Falkiewicz
 */
public abstract class SimpleTextWatcher implements TextWatcher {

    @Override
    public void beforeTextChanged(CharSequence s, int start, int count, int after) {
        // No action
    }

    @Override
    public void onTextChanged(CharSequence s, int start, int before, int count) {
        // No action
    }

    @Override
    public void afterTextChanged(Editable s) {
        // No action
    }
}

File: simplesearchview/src/main/java/com/ferfalk/simplesearchview/SimpleTextWatcher.kt
package com.ferfalk.simplesearchview

import android.text.Editable
import android.text.TextWatcher

/**
 * @author Fernando A. H. Falkiewicz
 */
abstract class SimpleTextWatcher : TextWatcher {
    override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {
        // No action
    }

    override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {
        // No action
    }

    override fun afterTextChanged(s: Editable) {
        // No action
    }
}